Gopkg.lock

```diff
@@ -1193,18 +1191,15 @@
     "github.com/spf13/pflag",
     "github.com/vmware/govmomi",
     "github.com/vmware/govmomi/find",
-    "github.com/vmware/govmomi/lookup/simulator",
```

Comment:
Removing these imports is causing CI to fail.

Reply:
Yup, was trying to removing unnecessary code but went a little too far it seems.